.github/workflows/ui-module-descriptor-generate.yml

```diff
@@ -0,0 +1,67 @@
+name: FOLIO UI Generate Module Descriptor
+
+on:
+  workflow_call:
+    inputs:
+      node-version:
+        required: true
+        type: string
+      folio-npm-registry:
+        required: true
+        type: string
+      package-version:
+        required: true
+        type: string
+
+jobs:
+  run:
+    name: Generate module descriptor
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout ${{ github.repository }}
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          show-progress: false # spammy
+
+      - name: Install Node ${{ inputs.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ inputs.node-version }}
+          check-latest: true
+          always-auth: true
+
+      - name: Set FOLIO NPM registry
+        run: yarn config set @folio:registry ${{ inputs.folio-npm-registry }}
+
+      - id: get-version
+        name: Get current @folio/stripes-cli version
+        # some packages may not specify @folio/stripes-cli as a dependency (e.g. stripes itself)
+        # in this case, we default to @latest
+        run: echo version=$(jq -r '.devDependencies["@folio/stripes-cli"]' package.json | sed s/null/latest/)
+
+      # we do it this way to avoid installing all modules
+      - name: Install @folio/stripes-cli
+        run: |-
+          rm package.json
+          yarn add @folio/stripes-cli@${{ steps.get-version.outputs.version }}
+          git checkout package.json
+
+      - name: Set package version number to ${{ inputs.package-version }}
+        run: |-
+          yarn config set version-git-tag false
+          yarn version --new-version ${{ inputs.package-version }}
+
+      - name: Generate module descriptor
+        run: yarn --silent stripes mod descriptor --full --strict | jq '.[]' > module-descriptor.json
```

Comment:
I think we can omit the "Install @folio/stripes-cli" step and change "Generate module descriptor" to
```
      - name: Generate module descriptor
        run: yarn --silent npx @folio/stripes-cli mod descriptor --full --strict | jq '.[]' > module-descriptor.json
```
From the [npx documentation](https://docs.npmjs.com/cli/v10/commands/npx):
> If any requested packages are not present in the local project dependencies, then they are installed to a folder in the npm cache, which is added to the PATH environment variable in the executed process. A prompt is printed (which can be suppressed by providing either --yes or --no).
> 
> Package names provided without a specifier will be matched with whatever version exists in the local project. Package names with a specifier will only be considered a match if they have the exact same name and version as the local dependency.


That feels cleaner. What do you think?

Reply:
I wanted to make sure we used the version of `@folio/stripes-cli` used in the package.json

Reply:
Agreed, and since with `npx` "Package names provided without a specifier will be matched with whatever version exists in the local project." that's what we'll get IIUC. How do you interpret this?

Reply:
ah i see, let me mess with it

Reply:
Alas, it appears yarn v1 is so very Very VERY old that `yarn npx` is not supported. I'd like to get this step cleaned up in a later iteration, but we don't have to worry about it here.